metadata.yaml

```diff
@@ -0,0 +1,10 @@
+# Copyright 2024 Canonical Ltd.
```

Comment:
the `metadata.yaml` is no longer needed since charmcraft 2.5 I believe the only file needed now is `charmcraft.yaml`

Reply:
Unfortunately we have to keep it due to https://github.com/canonical/data-platform-workflows/issues/169. I've started without, but CI job fail then.